mobile/src/home-view.js

```diff
@@ -426,9 +426,15 @@ class HomeView extends PureComponent {
   }
 
   selectSession = session => {
+    this.setState({ session, disable: false })
+    if (!this.state.questions.some(question => question.session === session.key)) {
```

Comment:
Will listeners for different sessions start to pile up?  Should be de-registering the existing session listeners?